src/lib/Controls.js

```diff
@@ -227,6 +230,22 @@ class Controls {
         return element;
     }
 
+    /**
+     * Add div for a group of controls
+     *
+     * @public
+     * @param {string} classNames - optional class names
+     * @return {HTMLElement} The created HTMLElement for a group of controls
+     */
+    addGroup(classNames = '') {
+        const group = document.createElement('div');
+        group.className = `bp-controls-group ${classNames}`;
```

Comment:
Minor: Use `classNames` util so that in case classNames aren't provided there won't be an empty space in the DOM

Reply:
We don't currently depend on that in preview, but we could add that and address it elsewhere in a follow up PR

Reply:
It is super minor and if we don't currently have it, no big deal 👍